lib/services.py

```diff
@@ -0,0 +1,14 @@
+from dataclasses import dataclass
+import os
+
+
+@dataclass
+class Services:
+    sensor_api_url: str
+    tomorrow_api_key: str
+
+
+S = Services(
+    sensor_api_url=os.getenv("SENSOR_API_URL", "http://galilei:8888/api/sensors"),
```

Comment:
Should default to localhost rather than galilei, maybe?

Reply:
If we use docker and have a fake sensor service then sure. Right now localhost would never work unless we're doing development on `galilei`.

Reply:
Could have no default, then?